### PR TITLE
Entries: document the ability to define a global normalizer

### PR DESCRIPTION
**Pull request by Arkni**
~~Please don't merge, the global normalizer is not released yet.~~

refs https://github.com/jquery-validation/jquery-validation/commit/a88ae88c6e11135961df630e97fbfbb00df90c35

### CODE REVIEW

File: entries/normalizer.xml [hunk 1]
```diff
@@ -71,5 +74,54 @@ $( "#myform" ).validate( {
...
+	<example>
+		<desc>Using a global normalizer in conjunction with a local one</desc>
+		<code><![CDATA[
+$( "#myform" ).validate( {
+  // This global normalizer will trim the value of all elements
+  // before validatng them.
+  normalizer: function( value ) {
+    return $.trim( value );
+  },
+	rules: {
```
Comment (Arkni) [inline entries/normalizer.xml]: Fix indentation

Comment (Arkni) [inline entries/normalizer.xml]: I should use either spaces or tabs

**Comment by Arkni** [comment]
I should also add the `normalizer` to `validate()` docs

File: entries/normalizer.xml [hunk 2]
```diff
@@ -38,21 +41,67 @@ $( "#myform" ).validate( {
...
+<html><![CDATA[
+<label for="url_input">url: </label>
+<input class="left" id="url_input" name="url_input">
+<br/>
+<input type="submit" value="Validate!">
+]]></html>
+	</example>
+
+	<example>
+		<desc>Using a global normalizer in conjunction with a local one</desc>
```
Comment (staabm) [inline entries/normalizer.xml]: Should we note the version in which this new api was added? (Not sure we do it in other places)

Comment (Arkni) [inline entries/normalizer.xml]: AFAIK, we never added the version to any of the available API, nor it was added before.
If you want I can add it, but that will be inconsistent with the rest of the documentation.

**Comment by staabm** [comment]
Ok, merge as is. Thx @arkni

**Pull request merged**